sahi/predict.py

```diff
@@ -767,7 +747,6 @@ def predict_fiftyone(
         category_mapping=model_category_mapping,
         category_remapping=model_category_remapping,
         load_at_init=False,
-        image_size=image_size,
```

Comment:
bu degisikligi de geri alalim

Reply:
Predict fonksiyonunu eski haline getirdim.

Reply:
@kadirnar henuz pushlamadin sanirim, hala ayni gorunuyor?

Reply:
Local de test işlemi gerçekleştirdiğim için biraz uzun sürdü.